docs/4.3/enterprise/workflow/ssh_approval_slack.md

```diff
@@ -0,0 +1,223 @@
+## Teleport Plugins Setup Quickstart
+
+If you're using Slack, you can be notified of [new teleport permission requests](https://gravitational.com/teleport/docs/cli-docs/#tctl-request-ls), approve or deny them on Slack with Teleport Slack Plugin. This guide covers it's setup.
```

Comment:
`new teleport permission requests` -> `new Teleport permission requests`

`covers it's setup` -> `covers its setup`

Reply:
I've rewritten these.